dbx_janitor_sqlendpoint.py

```diff
@@ -0,0 +1,68 @@
+#%%
+
+from dbclient import *
+
+print(dir())
+# python 3.6
+
+kill_endpoints = False
+stop_endpoints = True
+
+def cleanup_sqlendpoints(url, token, env_name):
+
+    sqlClient = SQLAnalyticsClient(token, url)
+    # Simple class to list versions and get active cluster list
+
+    # get the list of sql endpoints
+    terminate_list = sqlClient.get_terminate_sqlendpoints()
+    for endpoint in terminate_list:
+        if endpoint['keep_alive']==True:
```

Comment:
we could simplify this if statement to a single conditional, `if (endpoint['keep_alive'] == True) or (endpoint['keep_until'] == True): pass`